### DISCUSSION TITLE
Qt6 Install

### DISCUSSION THREAD

**Discussion by Melio-1** [microsoft/vcpkg#34729] [Q&A]
Can i install Qt6 library with install Qt6webengine?

**Comment by RealLitb** [comment]
If you ask about linux.. I had it working, but you best need a machine with more than 32RAM, or you may be successful with 32GB if you use lld instead of ld.bdf or gold. 

In the end I disabled building the webengine part of qt6webengine and patched the portfile to only build qtpdf (which is in qt6webengine because it's based on Google's PDF library that they embed into chrome).

Check https://github.com/microsoft/vcpkg/issues/32303#issuecomment-1616637949

**Reply by Melio-1** [reply]
I'm windows user, but I've 32GB RAM,
I need to install Qt6 without Qt6WebEngine,
How to disable it?

**Reply by dg0yt** [reply]
Simply don't install port `qt6`.
Install only the qt6 ports you really need.

**Reply by Melio-1** [reply]
How can i do it?
do mean install some dependencies of qt6 without install hole library?

**Reply by BillyONeal** [reply]
In your `vcpkg.json`, use `"dependencies": ["qt-webengine"]` rather than `"dependencies": ["qt"]`, or on the command line say `vcpkg install qt-webengine` rather than `vcpkg install qt`
